### ISSUE TITLE
Unable to read data using odbc::dbConnect connection

### ISSUE THREAD

**Issue by sccmckenzie** [r-dbi/odbc#252]
### Issue Description and Expected Result
I am unable to query data from any table in my Oracle connection. dbExistsTable and dbListTables confirm R recognizes the existence of tables, but any function that passes a SQL data query results in a "HY092" error.

Most recent version of R, DBI, and odbc are installed. CRAN and github versions of odbc show same results.

Other applications (Oracle SQL Developer, Microsoft Excel) have no issue querying data from database tables. 

### Database
```r
$dbms.name
[1] "Oracle"
$db.version
[1] "00.00.0000 Oracle Database 11g Enterprise Edition Release 11.2.0.4.0 - 64bit Prod"
$drivername
[1] "msorcl32.dll"
$odbc.version
[1] "03.80.0000"
$driver.version
[1] "06.01.7601"
$odbcdriver.version
[1] "02.50"
```
### Reproducible Example
Example:
```r
library(DBI)
library(odbc)

con <- dbConnect(odbc(),
                 dsn = "SAM",
                 uid = "user",
                 pwd = "password")

dbExistsTable(con,"MI_TABLE")
#>[1] TRUE
dbReadTable(con, "MI_TABLE")
#>Error: <SQL> 'SELECT * FROM "MI_TABLE"'
#>nanodbc/nanodbc.cpp:1581: HY092: [Microsoft][ODBC Driver Manager] Option type out of range
```
Other functions that send queries show similar results
```r
dbSendQuery(con,"SELECT * FROM MI_TABLE")
#>Error: <SQL> 'SELECT * FROM MI_TABLE'
#>nanodbc/nanodbc.cpp:1581: HY092: [Microsoft][ODBC Driver Manager] Option type out of range
tbl(con,"MI_TABLE")
#>Error: <SQL> 'SELECT *
#>FROM ("MI_TABLE") "zzz4"
#>WHERE (0 = 1)'
#>nanodbc/nanodbc.cpp:1581: HY092: [Microsoft][ODBC Driver Manager] Option type out of range 
```

<details>
<summary>Session Info</summary>

```r
devtools::session_info()
- Session info -----------------------------------------------------------------------------------------
 setting  value                       
 version  R version 3.5.2 (2018-12-20)
 os       Windows 7 x64 SP 1          
 system   i386, mingw32               
 ui       RStudio                     
 language (EN)                        
 collate  English_United States.1252  
 ctype    English_United States.1252  
 tz       America/Chicago             
 date     2019-01-20                  

- Packages ---------------------------------------------------------------------------------------------
 package     * version date       lib source        
 assertthat    0.2.0   2017-04-11 [1] CRAN (R 3.5.2)
 backports     1.1.3   2018-12-14 [1] CRAN (R 3.5.2)
 bindr         0.1.1   2018-03-13 [1] CRAN (R 3.5.2)
 bindrcpp      0.2.2   2018-03-29 [1] CRAN (R 3.5.2)
 bit           1.1-14  2018-05-29 [1] CRAN (R 3.5.2)
 bit64         0.9-7   2017-05-08 [1] CRAN (R 3.5.2)
 blob          1.1.1   2018-03-25 [1] CRAN (R 3.5.2)
 broom         0.5.1   2018-12-05 [1] CRAN (R 3.5.2)
 callr         3.1.1   2018-12-21 [1] CRAN (R 3.5.2)
 cellranger    1.1.0   2016-07-27 [1] CRAN (R 3.5.2)
 cli           1.0.1   2018-09-25 [1] CRAN (R 3.5.2)
 colorspace    1.4-0   2019-01-13 [1] CRAN (R 3.5.2)
 crayon        1.3.4   2017-09-16 [1] CRAN (R 3.5.2)
 DBI         * 1.0.0   2018-05-02 [1] CRAN (R 3.5.2)
 dbplyr      * 1.3.0   2019-01-09 [1] CRAN (R 3.5.2)
 desc          1.2.0   2018-05-01 [1] CRAN (R 3.5.2)
 devtools      2.0.1   2018-10-26 [1] CRAN (R 3.5.2)
 digest        0.6.18  2018-10-10 [1] CRAN (R 3.5.2)
 dplyr       * 0.7.8   2018-11-10 [1] CRAN (R 3.5.2)
 forcats     * 0.3.0   2018-02-19 [1] CRAN (R 3.5.2)
 fs            1.2.6   2018-08-23 [1] CRAN (R 3.5.2)
 generics      0.0.2   2018-11-29 [1] CRAN (R 3.5.2)
 ggplot2     * 3.1.0   2018-10-25 [1] CRAN (R 3.5.2)
 glue          1.3.0   2018-07-17 [1] CRAN (R 3.5.2)
 gtable        0.2.0   2016-02-26 [1] CRAN (R 3.5.2)
 haven         2.0.0   2018-11-22 [1] CRAN (R 3.5.2)
 hms           0.4.2   2018-03-10 [1] CRAN (R 3.5.2)
 httr          1.4.0   2018-12-11 [1] CRAN (R 3.5.2)
 jsonlite      1.6     2018-12-07 [1] CRAN (R 3.5.2)
 lattice       0.20-38 2018-11-04 [1] CRAN (R 3.5.2)
 lazyeval      0.2.1   2017-10-29 [1] CRAN (R 3.5.2)
 lubridate     1.7.4   2018-04-11 [1] CRAN (R 3.5.2)
 magrittr      1.5     2014-11-22 [1] CRAN (R 3.5.2)
 memoise       1.1.0   2017-04-21 [1] CRAN (R 3.5.2)
 modelr        0.1.2   2018-05-11 [1] CRAN (R 3.5.2)
 munsell       0.5.0   2018-06-12 [1] CRAN (R 3.5.2)
 nlme          3.1-137 2018-04-07 [1] CRAN (R 3.5.2)
 odbc        * 1.1.6   2019-01-20 [1] local         
 pillar        1.3.1   2018-12-15 [1] CRAN (R 3.5.2)
 pkgbuild      1.0.2   2018-10-16 [1] CRAN (R 3.5.2)
 pkgconfig     2.0.2   2018-08-16 [1] CRAN (R 3.5.2)
 pkgload       1.0.2   2018-10-29 [1] CRAN (R 3.5.2)
 plyr          1.8.4   2016-06-08 [1] CRAN (R 3.5.2)
 prettyunits   1.0.2   2015-07-13 [1] CRAN (R 3.5.2)
 processx      3.2.1   2018-12-05 [1] CRAN (R 3.5.2)
 ps            1.3.0   2018-12-21 [1] CRAN (R 3.5.2)
 purrr       * 0.2.5   2018-05-29 [1] CRAN (R 3.5.2)
 R6            2.3.0   2018-10-04 [1] CRAN (R 3.5.2)
 Rcpp          1.0.0   2018-11-07 [1] CRAN (R 3.5.2)
 readr       * 1.3.1   2018-12-21 [1] CRAN (R 3.5.2)
 readxl        1.2.0   2018-12-19 [1] CRAN (R 3.5.2)
 remotes       2.0.2   2018-10-30 [1] CRAN (R 3.5.2)
 rlang         0.3.1   2019-01-08 [1] CRAN (R 3.5.2)
 rprojroot     1.3-2   2018-01-03 [1] CRAN (R 3.5.2)
 rstudioapi    0.9.0   2019-01-09 [1] CRAN (R 3.5.2)
 rvest         0.3.2   2016-06-17 [1] CRAN (R 3.5.2)
 scales        1.0.0   2018-08-09 [1] CRAN (R 3.5.2)
 sessioninfo   1.1.1   2018-11-05 [1] CRAN (R 3.5.2)
 stringi       1.2.4   2018-07-20 [1] CRAN (R 3.5.2)
 stringr     * 1.3.1   2018-05-10 [1] CRAN (R 3.5.2)
 tibble      * 2.0.1   2019-01-12 [1] CRAN (R 3.5.2)
 tidyr       * 0.8.2   2018-10-28 [1] CRAN (R 3.5.2)
 tidyselect    0.2.5   2018-10-11 [1] CRAN (R 3.5.2)
 tidyverse   * 1.2.1   2017-11-14 [1] CRAN (R 3.5.2)
 usethis       1.4.0   2018-08-14 [1] CRAN (R 3.5.2)
 withr         2.1.2   2018-03-15 [1] CRAN (R 3.5.2)
 xml2          1.2.0   2018-01-24 [1] CRAN (R 3.5.2)
 yaml          2.2.0   2018-07-25 [1] CRAN (R 3.5.2)
```
</details>

**Comment by rozsoma** [comment]
I have exactly the same problem.

**Comment by matacarlos01** [comment]
Same issue here. Anyone found a solution?

**Comment by shrektan** [comment]
Have you tried using "select * from schema.table"? Will it give you different results?

Personally I use the ROracle package for the Oracle DB. It's not easy to install (you have to config the system ENV first) but it's fast and stable since it's developed by Oracle itself.

**Comment by jimhester** [comment]
This should have been fixed in [odbc 1.2.2](https://github.com/r-dbi/odbc/releases/tag/v1.2.2) you seem to be using an older version.
